docs/xshareclasses.md

```diff
@@ -588,6 +588,12 @@ behavior, which can improve the performance of class loading from the shared cla
 
 : Gives detailed output on the cache I/O activity, listing information about classes that are stored and found. Each class loader is given a unique ID (the bootstrap loader is always 0) and the output shows the class loader hierarchy at work, where class loaders must ask their parents for a class before they can load it themselves. It is typical to see many failed requests; this behavior is expected for the class loader hierarchy.
 
+### `portable`
+
+        -Xshareclasses:portable
+
+:   Enables AOT code to be compiled based on a chosen set of processor features that maximizes its portability across machines. It is currently only supported on x86.
```

Comment:
I am not a native English speaker, but "It is currently supported only on x86" sounds better to me.
Also, is support enabled for 32-bit as well? I guess so, but if not we may want to qualify further.
Also, maybe it's a good idea to specify the chosen processor features (features present in the Intel Sandy Bridge processors)

Reply:
Do you think we should spell out all the features or just state "features present in Intel Sandy Bridge"?

Reply:
I think users will be more familiar with the SandyBridge name than with each individual feature. It also gives them a simple reference point.